bin/extract_api_surface.py

```
@@ -0,0 +1,207 @@
+        "node_methods": {},
+        "mesh_interface_methods": {},
+        "top_level_exports": _get_top_level_exports(pkg_dir),
+    }
+
+    for cls in classes:
+        key = f"{cls.lower().replace('meshinterface', 'mesh_interface')}_methods"
+        if cls == "MeshInterface":
+            key = "mesh_interface_methods"
+        elif cls == "Node":
+            key = "node_methods"
+        for _src, (tree, _mod) in module_map.items():
```

Ruff (0.15.7)

[warning] When using only the values of a dict use the `values()` method

Replace `.items()` with `.values()`

(PERF102)